Gruntfile.js

```diff
@@ -56,51 +59,59 @@ module.exports = function( grunt ) {
     concat: {
       // JS Files
       // "dist/suite.js": [ "lib/qunit-custom.js", "lib/compat.js", "<file_strip_banner:lib/h.js>" ],
-      "dist/runner.js": [ "lib/console.js", "lib/qunit-custom.js", "lib/compat.js" ],
-      "dist/h.js": [ "<banner>", "<file_strip_banner:lib/h.js>", "lib/ring.js" ],
-      "dist/ringmark.js": [
-        "lib/archaic.js", "lib/console.js", "lib/qunit-custom.js",
-        "lib/compat.js",
-        "<banner>", "<file_strip_banner:lib/h.js>", "lib/ring.js",
-        "lib/h-logging.js"
-      ],
-      "dist/application.js": [
-        "dist/lodash.js",
-        "lib/jquery-1.7.2.js",
-        "lib/jquery.details.js",
-        "lib/app.js",
-
-        // Entries in App.Cache:
-        "dist/features.js",
-        "dist/fixtures.js",
-        "dist/apptypes.js",
-        "dist/browserscopekeys.js",
-        "dist/ringheaders.js",
-        // ...end
-
-        "lib/browserscope.js",
-        "lib/site.js"
-      ],
-      "dist/performance.js": [
-        "performance/src/moxie.js",
-        "performance/src/raf.js",
-        "performance/src/sprite.js"
-      ],
-
-      // HTML Compiled Fixtures
-      // "dist/fixtures.html": [ "tests/!(boilerplate)**/*fixture.html" ],
-
-      "index.html": [ "site/header.html", "site/footer.html" ]
+      dist: {
+        files: {
+          "dist/runner.js": [ "lib/console.js", "lib/qunit-custom.js", "lib/compat.js" ],
+          "dist/h.js": [ "<banner>", "lib/h.js", "lib/ring.js" ],
+          "dist/ringmark.js": [
+            "<banner>", "lib/h.js", "lib/ring.js",
+            "lib/archaic.js", "lib/console.js", "lib/qunit-custom.js",
+            "lib/compat.js",
+            "lib/h-logging.js"
+          ],
+          "dist/application.js": [
+            "dist/lodash.js",
+            "lib/jquery-1.7.2.js",
+            "lib/jquery.details.js",
+            "lib/app.js",
+
+            // Entries in App.Cache:
+            "dist/features.js",
+            "dist/fixtures.js",
+            "dist/apptypes.js",
+            "dist/browserscopekeys.js",
+            "dist/ringheaders.js",
+            // ...end
+
+            "lib/browserscope.js",
+            "lib/site.js"
+          ],
+          "dist/performance.js": [
+            "performance/src/moxie.js",
+            "performance/src/raf.js",
+            "performance/src/sprite.js"
+          ],
+
+          // HTML Compiled Fixtures
+          // "dist/fixtures.html": [ "tests/!(boilerplate)**/*fixture.html" ],
```

Comment:
Should we keep this? Since these files are under a subconfig `concat:dist`, if I recall this grunt behavior correctly, you could move this to say `concat:fixtures` and make the default tasks refer to `concat:dist` instead of `concat` to only run those concats.

Reply:
This line is already commented out on the main branch.